14_functions_lab/04.py

```diff
@@ -1,6 +1,13 @@
-"""
-Write a function that takes minlen and
-a list of words, and returns only the words
-longer than minlen
-"""
-
+
+import sys
+import os
+
+def length_validation(length, *words):
+	try:

```

Comment:
Catching an exception and printing an error message is called swallowing the exception, because now your program will keep going even though the function `length_validation` didn't do what it needed to do, and returned the value `None` instead of a list.

Swallowing exceptions may result in strange bugs, if a user misses the printed error message (since many programs print far more than one or two result lines)

It's unhealthy to swallow exceptions. Let them roll or catch them if you have a useful way to handle them